content/articles/java-program-to-store-unicode-characters-using-character-literals/index.md

```diff
@@ -0,0 +1,129 @@
+### Introduction
+Meaningless data elements are called `literals` because they have fixed values. Java literals are hence also known as constants. It is possible to represent a wide variety of values with the help of literals. Strings in Java are objects.
+Only a single String object may be stored in memory by Java's virtual engine if we specify the same string in many variables.
+There is an international standard for encoding characters that can be used by everyone. 
+
+Many different characters can be represented in various forms, such as text files and websites, using this character set. Characters can be stored in 4 bytes in Unicode. Character literals can be stored in char data types. You can pick between three alternative character encodings: UTF-8, 16 and 32. Web content should use only UTF-8.
+### Method approaches
+1. First method: Giving the char datatypes Unicode.

```

Comment:
I suggest you make this a subheading (using #### ) and also change it a little bit to avoid repetition.
ie : "  #### 1. Giving the char datatypes Unicode       "

Do the same for the second and third approaches